myhoard/util.py

```diff
@@ -569,3 +569,15 @@ def run(self):
                 self.stats.unexpected_exception(ex=exception, where="RateTracker.run")
                 self.stats.increase("myhoard.ratetracker.errors")
                 self._reset()
+
+
+class SlaveStatus(TypedDict):
+
+    Relay_Log_File: str
+    Slave_SQL_Running: str
+    Slave_SQL_Running_State: str
+
+
+def get_slave_status(cursor) -> Optional[SlaveStatus]:
+    cursor.execute("SHOW SLAVE STATUS")
```

Comment:
Can you add a comment about why this can be None here ? (ie. it's not just the typing of cursor that is inconvenient for us, it's a thing that happens for real when the replication was never configured).

Reply:
Done